src/data/opposite.lean

```diff
@@ -58,14 +58,16 @@ def unop : αᵒᵖ → α := id
 lemma op_injective : function.injective (op : α → αᵒᵖ) := λ _ _, id
 lemma unop_injective : function.injective (unop : αᵒᵖ → α) := λ _ _, id
 
-@[simp] lemma op_inj_iff (x y : α) : op x = op y ↔ x = y := iff.rfl
-@[simp] lemma unop_inj_iff (x y : αᵒᵖ) : unop x = unop y ↔ x = y := iff.rfl
-
 @[simp] lemma op_unop (x : αᵒᵖ) : op (unop x) = x := rfl
 @[simp] lemma unop_op (x : α) : unop (op x) = x := rfl
 
 attribute [irreducible] opposite
 
+-- We could prove these by `iff.rfl`, but that would make these elegible for `dsimp`. That would be
```

Comment:
eligible

Reply:
Yeah, I saw that but didn't want to have to wait for a new cache. Will edit before merging.